### ISSUE TITLE
generate function signature listing from markdown (stan-functions.txt)

### ISSUE THREAD

**Issue by mitzimorris** [stan-dev/docs#179]
#### Summary:

Generate a listing of functions signatures from the documentation for use with RStan and Stan emacs mode via a python script (roughly) along the lines of existing build scripts `add_redirects.py`, `link_to_latest.py`.

#### Description:

The index of the functions reference pdf was used as the basis of the file `stan-functions.txt`; when moving to markdown, html comment tags were added which captured the equivalent information in the markdown files, e.g.:

```
> grep '<!-- ' src/functions-reference/covariance_matrix_distributions.Rmd 
<!-- real; wishart ~; -->
<!-- real; wishart_lpdf; (matrix W | real nu, matrix Sigma); -->
<!-- matrix; wishart_rng; (real nu, matrix Sigma); -->
<!-- real; inv_wishart ~; -->
<!-- real; inv_wishart_lpdf; (matrix W | real nu, matrix Sigma); -->
<!-- matrix; inv_wishart_rng; (real nu, matrix Sigma); -->
```

#### Additional Information:

Soliciting input as to most useful file format from community via Discourse.

#### Current Version:
v2.18.0

**Comment by mitzimorris** [comment]
@bgoodri - would prefer to do this instead of https://github.com/stan-dev/docs/pull/128 - is this OK with you?

**Comment by bgoodri** [comment]
I think so

**Comment by WardBrian** [comment]
This was resolved by #180
